### ISSUE TITLE
Loss evolution for pre-trained models?

### ISSUE THREAD

**Issue by vatsalaggarwal** [ebadawy/voice_conversion#16]
Hey great work! Just wondering any chance you would have the loss evolution for your pretrained models?

**Comment by ebadawy** [comment]
i don't have that at the moment. will try to upload it if i get to train the model again.
